src/coreclr/scripts/superpmi.py

```diff
@@ -2100,7 +2117,7 @@ def main(args):
     # await/async requires python >= 3.5
```

Comment:
Do you mind changing the comment to >= 3.8

Reply:
Do you want to force 3.8+? I made the comment say 3.8, but I'm not sure it actually requires anything more than 3.5 (you said it needs that for async)

Reply:
It technically requires 3.7 as 3.6 on windows has an async bug.